src/index.ts

```diff
@@ -79,20 +80,68 @@ export class Aracari<T extends HTMLElement = HTMLElement> {
     return this.getNodeByAddress(address);
   }
 
+  private nonLatinMatchAndReplacement = ({
+    node,
+    text,
+    nodes,
+    replacementIndex
+  }: {
+    node: any;
+    text: string,
+    nodes: T | Node | (T | Node)[];
+    replacementIndex: number;
+  }) => {
+    // Using this pattern because look-around regex not supported in some browsers
+    // (\b) word bound does not work with non-latin scripts
+    const pattern = new RegExp(
+      `(?:^|\\s)${escapeRegExp(text)}(?:$|\\s|[.!?,"'])`,
```

Comment:
woah

Reply:
Yezzir on the Todo just want to make sure I can get this in sooner rather than later.